specification/applicationinsights/data-plane/readme.md

```diff
@@ -100,12 +100,12 @@ directive:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-applicationinsights/azure/applicationinsights
+  output-folder: $(python-sdks-folder)/azure-applicationinsights-query
```

Comment:
Should be output-folder: $(python-sdks-folder)/azure-applicationinsights-query/azure/applicationinsights/query